pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -244,8 +244,12 @@ public void start() throws Exception {
 
   public void stop() throws Exception {
     LOGGER.info("Shutting down Network !!");
-
     _state.set(State.SHUTTING_DOWN);
+    try {
+      Thread.sleep(10000);
```

Comment:
Broker will continue to server requests for 10seconds in this case ? Will this addressing the issue in that case ?

It also seems unrelated to the main PR. Move to a separate PR